content/docs/reconciliation.md

```diff
@@ -69,7 +69,7 @@ After handling the DOM node, React then recurses on the children.
 
 ### Component Elements Of The Same Type
 
-When a component updates, the instance stays the same, so that state is maintained across renders. React updates the props of the underlying component instance to match the new element, and calls `componentWillReceiveProps()` and `componentWillUpdate()` on the underlying instance.
+When a component updates, the instance stays the same, so that state is maintained across renders. React then updates the props of the underlying component instance to match the new element. If the component implements the `getDerivedStateFromProps()` static method, React will similarly update the component instance with any changed state.
```

Comment:
I'm worried this will encourage people to implement it, which we usually *don't* want. Let's just drop this sentence?